tests/conftest.py

```diff
@@ -67,17 +57,37 @@ def mock_product_test_spotlight(
         constraints=MyProductConstraints,
         opportunity_properties=MyOpportunityProperties,
         order_parameters=MyOrderParameters,
-        backend=product_backend,
+        create_order=mock_create_order,
+        search_opportunities=mock_search_opportunities,
     )
 
 
 @pytest.fixture
 def stapi_client(
-    root_backend, mock_product_test_spotlight, base_url: str
+    mock_product_test_spotlight,
+    base_url: str,
+    orders_db: InMemoryOrderDB | None = None,
```

Comment:
I think I may have misled you here a little. I'm realizing if nothing needs to pass in a database instance via a fixture of this name then you can remove this and just do `InMemoryOrderDB()` below where you need one instead of making it conditional.